src/ast/helpers.rs

```diff
@@ -225,6 +225,15 @@ pub fn is_super_call_with_arguments(func: &Expr, args: &[Expr]) -> bool {
     }
 }
 
+/// Format the module name for a relative import.
+pub fn format_relative_import(level: &Option<usize>, module: &Option<String>) -> String {
```

Comment:
From Rust API point of view, I think it's kinda odd to pass around `&Option<T>`, `Option<&T>` is much more natural. You can get a `Option<&T>` via [`Option::as_ref()`](https://doc.rust-lang.org/std/option/enum.Option.html#method.as_ref).

And to get a `Option<&str>` from a `&Option<String>` you can use [Option::as_deref()](https://doc.rust-lang.org/std/option/enum.Option.html#method.as_deref).

Reply:
Oh, thank you! That's really helpful feedback. I'm gonna fix this everywhere.